service/src/main/kotlin/vdi/model/DBPlatform.kt

```diff
@@ -1,6 +1,16 @@
 package vdi.model
 
+import org.veupathdb.vdi.lib.common.util.or
+
 enum class DBPlatform(val value: String) {
   Oracle("Oracle"),
-  Postgres("Postgres"),
+  Postgres("Postgresql");
+
+  companion object {
+    fun fromPlatformString(platformString: String?): DBPlatform {
+      return values().asSequence()
```

Comment:
`values().asSequence()` can be replaced with `entries`.

As of Kotlin 1.9, use of `values()` is deprecated, and the `find` mixin for `Iterable` avoids the setup work involved in creating a `Sequence` instance.